src/app/containers/SwapHistory/index.tsx

```diff
@@ -219,16 +257,24 @@ function AssetRow({ data, itemFrom, itemTo }: AssetProps) {
       <td>
         <div className="d-flex align-items-center justify-content-between col-lg-10 col-md-12 p-0">
           <div>
-            {tx && <p className="m-0">Confirmed</p>}
-            {!tx && <p className="m-0">Failed</p>}
+            {!data.status && <p className="m-0">Confirmed</p>}
+            {data.status === TxStatus.FAILED && <p className="m-0">Failed</p>}
+            {data.status === TxStatus.PENDING && <p className="m-0">Pending</p>}
```

Comment:
Add translations to this, I think state translation keys are defined in common group